File: sqry-lang-javascript/tests/fixtures/localvars/basic.js
// Basic variable declaration + usage tests

function simpleVar() {
    let x = 10;
    let y = x + 1;
    return y;
}

function constVar() {
    const count = 42;
    console.log(count);
}

function multipleDeclarators() {
    let a = 1, b = 2;
    let c = a + b;
    return c;
}

function paramRef(name, age) {
    const result = name;
    console.log(age);
    return result;
}

File: sqry-lang-rust/tests/fixtures/localvars/basic.rs
// Basic variable declaration + usage tests

fn simple_var() {
    let x = 10;
    let y = x + 1;
    println!("{}", y);
}

fn const_binding() {
    let count = 42;
    println!("{}", count);
}

fn mutable_var() {
    let mut x = 10;
    x += 1;
    println!("{}", x);
}

fn param_ref(name: &str, age: u32) {
    let result = name;
    println!("{} {}", result, age);
}
